modules/lightning_features/lightning_media/lightning_media.module

```diff
@@ -25,15 +29,63 @@ use Drupal\media_entity\MediaInterface;
 use Drupal\media_entity\MediaBundleInterface;
 
 /**
- * Implements hook_field_widget_info_alter().
+ * Implements hook_field_widget_third_party_settings_form().
  */
-function lightning_media_field_widget_info_alter(array &$info) {
-  if (isset($info['entity_browser_entity_reference'])) {
-    Override::pluginClass($info['entity_browser_entity_reference'], EntityReferenceBrowserWidget::class);
+function lightning_media_field_widget_third_party_settings_form(WidgetInterface $widget) {
```

Comment:
All of these changes constitute a major refactor of how we were customizing the behavior of image widgets. This is a *far* more stable way to achieve what we were previously doing with plugin hijacking.